Source/ACE/Managers/GuidManager.cs

```diff
@@ -164,5 +169,23 @@ public static ObjectGuid NewItemGuid()
         {
             return new ObjectGuid(itemAlloc.Alloc());
         }
+
+        /// <summary>
+        /// Returns Most Recently assigned Guid for Items / Player Items, to be used only for informational purposes.
+        /// </summary>
+        /// <returns></returns>
+        public static ObjectGuid CurrentItemGuid()
```

Comment:
Ensuring these functions are not "accidentally" used incorrectly is going to be up to all who reviews the PRs. The summary clearly states they are to be only used for informational purposes. Outside of the idlist command, the functions will in all likelihood not be referenced again.